apps/api/src/app/gene/gene.service.ts

```diff
@@ -0,0 +1,36 @@
+import {Injectable, OnModuleInit} from '@nestjs/common';
+import {PrismaService} from "../prisma.service";
+
+@Injectable()
+export class GeneService {
+  constructor(private prismaService: PrismaService) {}
+
+  async fetchGeneSymbols() {
```

Comment:
I think you have being use HugoSymbols through out this pr, maybe stick to that?

Reply:
changed fetchGeneSymbols to fetchHugoSymbols